src/System.Text.Json/tests/Utf8JsonWriterTests.cs

```diff
@@ -2494,6 +2500,132 @@ public void WritingTooLargeBase64Bytes(bool formatted, bool skipValidation)
             }
         }
 
+        [ConditionalTheory(nameof(IsX64))]
+        [OuterLoop]
+        [InlineData(true, true)]
+        [InlineData(true, false)]
+        [InlineData(false, true)]
+        [InlineData(false, false)]
+        public void WritingLargestPossibleBase64Bytes(bool formatted, bool skipValidation)
+        {
+            byte[] value;
+
+            try
+            {
+                value = new byte[125_000_000];
+            }
+            catch (OutOfMemoryException)
+            {
+                return;
```

Comment:
This is flaky, no?

Reply:
We have this pattern elsewhere in the code base, but you bring a good point. We can exclude this test on linux. It should be deterministic for other OSes.